esx_service/vmci/vmci_client.c

```diff
@@ -349,5 +366,22 @@ Vmci_GetReply(int port, const char* json_request, const char* be_name,
    req.mlen = strnlen(json_request, MAXBUF) + 1;
    req.msg = json_request;
 
+   ans->errBuf = malloc(512);
```

Comment:
Using an array of 512 chars and a pointer field that's initialized to the array ( to avoid GO side jugglery to get a pointer to the C array).